index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Global school home's page</title>
+  <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/mobile_style.css" />
+  <link rel="stylesheet" href="css/Destop_style.css" />
+</head>
+
+<body>
+  <header class="dark_bar">
+    <ul>
+      <li>
+        <a href="#"><i class="fab fa-facebook-f"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-twitter"></i></a>
+      </li>
+      <li><a href="#">English</a></li>
+      <li><a href="#">My Page</a></li>
+      <li><a href="#">Logout</a></li>
+    </ul>
+  </header>
+  <div class="white_bar">
+    <div class="logo">
+      <a href=""><span>GLO</span>BAL SCH<span>OOL</span></a>
+    </div>
+    <nav class="Lnav">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="camp">Campaign</li>
+      </ul>
+      <div class="menu_bar">
+        <i class="fas fa-bars"></i>
+      </div>
```

Comment:
- Great job for adding beautiful animation for your hamburger 👍 I think it would be better if you fill the page on the mobile menu to have a great user experience 👍
![2023-04-25_13-40](https://user-images.githubusercontent.com/52010789/234261246-95e1e583-d66e-48ec-944d-4bed17c40397.png)